packages/@okta/vuepress-site/docs/reference/api/brands/index.md

```diff
@@ -1180,6 +1180,8 @@ The Theme object defines the following properties:
 | `errorPageTouchPointVariant`          | Enum     | Variant for Error page. Accepted values: `OKTA_DEFAULT`, `BACKGROUND_SECONDARY_COLOR`, `BACKGROUND_IMAGE`.                 | `OKTA_DEFAULT`    |
 | `emailTemplateTouchPointVariant`      | Enum     | Variant for Email templates. Accepted values: `OKTA_DEFAULT`, `FULL_THEME`.                                                | `OKTA_DEFAULT`    |
 
+> **Note:** Contrast color will be used by pages to identify opacity of text color where primary or secondary color is used as background color.
```

Comment:
@TiffanyNg-okta @francoislehoux-okta forgot to add about contrast colors. please verify if this looks accurate

Reply:
Hmm what were you trying to say with `identify opacity of text color`? Since contrast color is just a color, we should be comparing/using it against a color. So I think we can just say `determine the text color` or something similar. Feels like including `opacity` might be too much detail that doesn't make sense without more explanation/context. Another option might be `identify the shade of text color`. 

But also, I'm not a writer so 🤷 I think you've got the general idea though so it's fine with me if everyone else is ok with it.

Reply:
I am also not sure how to represent this piece. I like `identify the shade of text color`, will update with that to start with.

Reply:
I prefer `opacity`, since `shade` could mean hue. How about:

"Contrast color will be used by pages to optimize the opacity of text color when primary or secondary color is used as the background."